AUTHORS

```diff
@@ -410,6 +410,7 @@ Shaun McVey <shaun@cockroachlabs.com>
 Shawn Morel <shawn@squareup.com> <shawn@strangemonad.com> <strangemonad@squareup.com>
 Sherman Grewal <sherman.grewal@cockroachlabs.com> <grewalsherman@gmail.com>
 Shiranka Miskin <shiranka@cockroachlabs.com> <shiranka.miskin@gmail.com>
+Shivam Saraf <shivam.saraf@cockroachlabs.com>
```

Comment:
If you want feel free to an additional email address like other folks have done, up to you.